config/config/sc-config.yaml

```diff
@@ -232,6 +232,7 @@ prometheus:
         limits:
           cpu: 10m
           memory: 100Mi
+  predictLinearLimit: 66
```

Comment:
Could you add a comment above explaining this config parameter? I'm sure I'll forget next week. :smile: